### PR TITLE
docs: keep Python COC, update README

### PR DESCRIPTION
**Pull request by dandhlee**
Before you open a pull request, note that this repository is forked from [here](https://github.com/docascode/sphinx-docfx-yaml/).
Unless the issue you're trying to solve is unique to this specific repository, 
please file an issue and/or send changes upstream to the original as well.

__________________________________________________________________

updating README and removing one of the duplicate COC, keeping the one that's available in other Python repositories.

- [ ] Tests pass
- [X] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,7 +35,13 @@ Basic Workflow
...
 -------
 
-First you need to install docfx-yaml:
+To use this forked version, install GCP docfx-yaml:
+
+.. code:: bash
+
+    pip install gcp-sphinx-docfx-yaml
+
+To use the default original, install original docfx-yaml:
```
Comment (tbpg) [inline README.rst:44]: I think we can delete this and the command below.

**Review by tbpg — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (dandhlee) [reply README.rst:44]: did you mean just the second part of the original snippet? or the pip install for gcp-sphinx-docfx-yaml as well?

Comment (tbpg) [reply README.rst:44]: We don't need to say how to install the original. How to install this one is good to keep.

Comment (dandhlee) [reply README.rst:44]: Gotcha. Just got confused by the range of snippet shown on the front page D:

**Pull request merged**